GeneratedRegexPatterns/HighConfidenceSecurityModels.json

```diff
@@ -317,6 +317,24 @@
     "Name": "AzureContainerRegistryIdentifiableKey",
     "DetectionMetadata": "Identifiable"
   },
+  {
+    "Pattern": "(^|[^0-9a-z])(?<refine>oy2[a-p][0-9a-z]{15}[aq][0-9a-z]{11}[eu][bdfhjlnprtvxz357][a-p][0-9a-z]{11}[aeimquy4])([^aeimquy4]|$)",
+    "Id": "SEC101/030",
+    "Name": "NuGetApiKey",
```

Comment:
>NuGetApiKey [](http://example.com/codeflow?start=13&length=11)

It appears these patterns were added without regen'ing the rules. We should make the build script break if there are open edits after it executes (which could indicate this auto-generated file was updated and needs to be checked in).